[2022-12-16 10:56:24,969] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-12-16T00:47:00+00:00 [queued]>
[2022-12-16 10:56:25,449] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-12-16T00:47:00+00:00 [queued]>
[2022-12-16 10:56:25,450] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-16 10:56:25,470] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-16 10:56:25,470] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-16 10:56:26,166] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_last_id> on 2022-12-16 00:47:00+00:00
[2022-12-16 10:56:26,390] {standard_task_runner.py:52} INFO - Started process 2879 to run task
[2022-12-16 10:56:26,746] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'prepare_data', 'get_last_id', 'scheduled__2022-12-16T00:47:00+00:00', '--job-id', '1471', '--raw', '--subdir', 'DAGS_FOLDER/prepare_data.py', '--cfg-path', '/tmp/tmp20nijysi', '--error-file', '/tmp/tmpoq8gni2t']
[2022-12-16 10:56:26,805] {standard_task_runner.py:80} INFO - Job 1471: Subtask get_last_id
[2022-12-16 10:56:29,965] {task_command.py:369} INFO - Running <TaskInstance: prepare_data.get_last_id scheduled__2022-12-16T00:47:00+00:00 [running]> on host 8e2f65ec4c74
[2022-12-16 10:56:33,191] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=prepare_data
AIRFLOW_CTX_TASK_ID=get_last_id
AIRFLOW_CTX_EXECUTION_DATE=2022-12-16T00:47:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-16T00:47:00+00:00
[2022-12-16 10:56:33,612] {base.py:68} INFO - Using connection ID 'gp_conn' for task execution.
[2022-12-16 10:56:34,456] {python.py:173} INFO - Done. Returned value was: None
[2022-12-16 10:56:36,632] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=prepare_data, task_id=get_last_id, execution_date=20221216T004700, start_date=20221216T105624, end_date=20221216T105636
[2022-12-16 10:56:37,025] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-16 10:56:38,316] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
